Library/Formula/ghc.rb

```diff
@@ -10,163 +12,91 @@ class Ghc < Formula
     sha256 "ad654abb4c2459b6cc764f275d5b6b141669e366f5051c58d871e54cb71a250c" => :mountain_lion
   end
 
-  option "32-bit"
   option "with-tests", "Verify the build using the testsuite."
-
   deprecated_option "tests" => "with-tests"
 
-  # http://hackage.haskell.org/trac/ghc/ticket/6009
-  depends_on :macos => :snow_leopard
-  depends_on "gcc" if MacOS.version == :mountain_lion
```

Comment:
That ticket doesn't apply, and gcc on ML, definitely not. I've only been able to test down to 10.7, but I think it might work on 10.5 or 10.6 using llvm-gcc.